File: pizzeriaProject/pizzeriaApp/Models/Pizza.py
from django.db import models

from .Chef import Chef


class Pizza(models.Model):
    sort = models.CharField(max_length=50)
    sauce = models.CharField(max_length=50, default=None)
    price = models.FloatField()
    chef = models.ForeignKey(Chef, on_delete=models.CASCADE, related_name='pizzas')
    weight = models.IntegerField()
    calories = models.IntegerField()

    orders=models.ManyToManyField('Order',through='PizzaOrder')

    def __str__(self):
        return self.sort

File: front-endPizzeria/src/models/Pizza.ts
import { Chef } from "./Chef";
import { Order } from "./Order";

export interface Pizza{
    id?: number;
    sort:string;
    sauce:string;
    price:number;
    weight:number;
    chef?:number | Chef;
    orders?:Order[];
    calories:number;
    // chef_id?:number;
}